content/docs/collector/about.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Getting Started"
+weight: 1
+---
+
+## Introduction
+
+The OpenTelemetry Collector offers a vendor-agnostic implementation on how to
+receive, process, and export telemetry data. In addition, it removes the need to
+run, operate, and maintain multiple agents/collectors in order to support
+open-source observability data formats (e.g. Jaeger, Prometheus, etc.) sending
+to multiple open-source or commercial back-ends.
+
+Objectives:
+
+- Usable: Reasonable default configuration, supports popular protocols, runs and collects out of the box.
+- Performant: Highly stable and performant under varying loads and configurations.
+- Observable: An exemplar of an observable service.
+- Extensible: Customizable without touching the core code.
+- Unified: Single codebase, deployable as an agent or collector with support for traces, metrics, and logs (future).
+
+## Deployment
+
+The OpenTelemetry Collector consists of a single binary and two deployment methods:
+
+1. An agent running with the application or on the same host as the application
+(e.g. binary, sidecar, or daemonset)
+2. A collector running as a standalone service (e.g. container or deployment)
+
+### Agent
+
+It is recommended to deploy the Agent on every host within an environment. In
+doing so, the Agent is capable of adding metadata to observability data it
+receives or collects. In addition, the Agent can offload responsibilities that
+client instrumentation would otherwise need to handle including batching,
+retry, encryption, compression, and more.
+
+### Collector
+
+Optionally, a Collector can be deployed in every cluster / datacenter / region.
+The Collector runs as a standalone service and can offer advanced capabilities
+over the Agent including tail-based sampling. In addition, the Collectors can
+limit the number of egress points required to send data. Each Collector
+operates independently so it is easy to scale the architecture based on
+performance needs.
+
+## Getting Started
+
+### Demo
+
+Deploys a load generator, agent, and collector as well as Jaeger, Zipkin and
+Prometheus back-ends. More information can be found on the demo
+[README.md](https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples/demo)
+
+```bash
+$ svn export https://github.com/open-telemetry/opentelemetry-collector/trunk/examples/demo; \
```

Comment:
What does `svn export` do? Can SVN connect to github? Why not `git clone`?

Reply:
It downloads a specific file instead of the whole repository -- I am not aware of a better way to do this but open to suggestions. Cloning everything to get a file is not ideal (but neither is having a dependency on svn...)

Reply:
Agreed with Tigran - git commands are preferable to svn, even if it means downloading extra data. In my experience most demos require cloning an entire repository.

Reply:
Switched